### ISSUE TITLE
Commands are echoed without quotation marks in the log

### ISSUE THREAD

**Issue by dbohdan** [nat-n/poethepoet#169]
When you define a command task with parts in quotes, the command shows in the log without the quotes. This makes it harder to understand the log and also prevents you from copying and pasting the command into your shell.

For example, for the task

```toml
foo = "echo 'Hello, world!'"
```

the output from Poe the Poet 0.23.0 is 

```
> poe foo
Poe => echo Hello, world!
Hello, world!
```

But as a user, I expect output like

```
> poe foo
Poe => echo 'Hello, world!'
Hello, world!
```

I don't think the quotation marks must necessarily be those in the literal text. What I have done in similar situations is join the command with `shlex.join` before logging it.

**Comment by nat-n** [comment]
hi @dbohdan, thanks for the feedback.

I think your suggestion makes sense, I'll try get it into the next release.

**Comment by nat-n** [comment]
This enhancement is now available in [v0.24.0](https://pypi.org/project/poethepoet/0.24.0/) 🚀

**Comment by dbohdan** [comment]
That was quick! Thanks.
